evm-bytecode/examples/0xe592427a0aece92de3edee1f18e0157c05861564

```diff
@@ -0,0 +1,6306 @@
+DUP1
+DUP6
+MLOAD
+DUP1
+DUP4
+MSTORE
+PUSH1 0x40
+DUP7
+ADD
+SWAP2
+POP
+PUSH1 0x40
+DUP5
+DUP3
+MUL
+DUP8
+ADD
+ADD
+SWAP3
+POP
+DUP4
+DUP8
+ADD
+DUP6
+JUMPDEST
+DUP3
+DUP2
+LT
+ISZERO
+PUSH2 0x2d1b
+JUMPI
+PUSH32 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffc0
+DUP9
+DUP7
+SUB
+ADD
+DUP5
+MSTORE
+PUSH2 0x2d09
+DUP6
+DUP4
+MLOAD
+PUSH2 0x2b77
+JUMP
+JUMPDEST
+SWAP5
+POP
+SWAP3
+DUP6
+ADD
+SWAP3
+SWAP1
+DUP6
+ADD
+SWAP1
+PUSH1 0x01
+ADD
+PUSH2 0x2ccf
+JUMP
+JUMPDEST
+POP
+SWAP3
+SWAP8
+SWAP7
+POP
+POP
+POP
+POP
+POP
+POP
+POP
+JUMP
+JUMPDEST
+PUSH1 0x00
+PUSH1 0x20
+DUP3
+MSTORE
+PUSH2 0x26d0
+PUSH1 0x20
+DUP4
+ADD
+DUP5
+PUSH2 0x2b77
+JUMP
+JUMPDEST
+PUSH1 0x20
+DUP1
+DUP3
+MSTORE
+PUSH1 0x12
+SWAP1
+DUP3
+ADD
+MSTORE
+PUSH32 0x546f6f206d756368207265717565737465640000000000000000000000000000
+PUSH1 0x40
+DUP3
+ADD
+MSTORE
+PUSH1 0x60
+ADD
+SWAP1
+JUMP
+JUMPDEST
+PUSH1 0x20
+DUP1
+DUP3
+MSTORE
+PUSH1 0x13
+SWAP1
+DUP3
+ADD
+MSTORE
+PUSH32 0x546f6f206c6974746c6520726563656976656400000000000000000000000000
+PUSH1 0x40
+DUP3
+ADD
+MSTORE
+PUSH1 0x60
+ADD
+SWAP1
+JUMP
+JUMPDEST
+PUSH1 0x00
+PUSH1 0x20
+DUP3
+MSTORE
+DUP3
+MLOAD
+PUSH1 0x40
+PUSH1 0x20
+DUP5
+ADD
+MSTORE
+PUSH2 0x2dc5
+PUSH1 0x60
+DUP5
+ADD
+DUP3
+PUSH2 0x2b77
+JUMP
+JUMPDEST
+SWAP1
+POP
+PUSH20 0xffffffffffffffffffffffffffffffffffffffff
+PUSH1 0x20
+DUP6
+ADD
+MLOAD
+AND
+PUSH1 0x40
+DUP5
+ADD
+MSTORE
+DUP1
+SWAP2
+POP
+POP
+SWAP3
+SWAP2
+POP
+POP
+JUMP
+JUMPDEST
+SWAP1
+DUP2
+MSTORE
+PUSH1 0x20
+ADD
+SWAP1
+JUMP
+JUMPDEST
+PUSH1 0x00
+DUP1
+DUP4
+CALLDATALOAD
+PUSH32 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe1
+DUP5
+CALLDATASIZE
+SUB
+ADD
+DUP2
+SLT
+PUSH2 0x2e2e
+JUMPI
+DUP3
+DUP4
+REVERT
+JUMPDEST
+DUP4
+ADD
+DUP1
+CALLDATALOAD
+SWAP2
+POP
+PUSH8 0xffffffffffffffff
+DUP3
+GT
+ISZERO
+PUSH2 0x2e48
+JUMPI
+DUP3
+DUP4
+REVERT
+JUMPDEST
+PUSH1 0x20
+ADD
+SWAP2
+POP
+CALLDATASIZE
+DUP2
+SWAP1
+SUB
+DUP3
+SGT
+ISZERO
+PUSH2 0x2e5d
+JUMPI
+PUSH1 0x00
+DUP1
+REVERT
+JUMPDEST
+SWAP3
+POP
+SWAP3
+SWAP1
+POP
+JUMP
+JUMPDEST
+PUSH1 0x40
+MLOAD
+DUP2
+DUP2
+ADD
+PUSH8 0xffffffffffffffff
+DUP2
+GT
+DUP3
+DUP3
+LT
+OR
+ISZERO
+PUSH2 0x2e80
+JUMPI
+'fe'(Unknown Opcode)
+JUMPDEST
+PUSH1 0x40
+MSTORE
+SWAP2
+SWAP1
+POP
+JUMP
+JUMPDEST
+PUSH1 0x00
+PUSH8 0xffffffffffffffff
+DUP3
+GT
+ISZERO
+PUSH2 0x2e9c
+JUMPI
+'fe'(Unknown Opcode)
+JUMPDEST
+POP
+PUSH1 0x1f
+ADD
+PUSH32 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe0
+AND
+PUSH1 0x20
+ADD
+SWAP1
+JUMP
+JUMPDEST
+PUSH1 0x00
+JUMPDEST
+DUP4
+DUP2
+LT
+ISZERO
+PUSH2 0x2ee3
+JUMPI
+DUP2
+DUP2
+ADD
+MLOAD
+DUP4
+DUP3
+ADD
+MSTORE
+PUSH1 0x20
+ADD
+PUSH2 0x2ecb
+JUMP
+JUMPDEST
+DUP4
+DUP2
+GT
+ISZERO
+PUSH2 0x108f
+JUMPI
+POP
+POP
+PUSH1 0x00
+SWAP2
+ADD
+MSTORE
+JUMP
+JUMPDEST
+PUSH20 0xffffffffffffffffffffffffffffffffffffffff
+DUP2
+AND
+DUP2
+EQ
+PUSH2 0x2f16
+JUMPI
+PUSH1 0x00
+DUP1
+REVERT
+JUMPDEST
+POP
+JUMP
+'fe'(Unknown Opcode)
+LOG1
+PUSH5 0x736f6c6343
+STOP
+SMOD
+MOD
+STOP
+EXP
```

Comment:
It's probably too big example. Is it possible to decrease size?

Reply:
The example corresponds to a smart contract taken from the Ethereum blockchain and the bytecode of real-world smart contracts is usually very large. We are looking for a smaller example.

Reply:
Done in [b26e05e](https://github.com/antlr/grammars-v4/pull/2723/commits/b26e05e086b62ef419ac45a33fd5b42143123fba)